raspbian/README.md

```diff
@@ -18,3 +18,5 @@ Raspbian is a community project under active development, with an emphasis on im
 - [Applications](applications/README.md)
     - [Camera](applications/camera.md)
     - [OMXPlayer](applications/omxplayer.md)
+    - [vcgencmd](applications/vcgencmd.md)
+    - [vcdbg](applications/vcdbg.md)
```

Comment:
Should `tvservice` be included in this list?

Reply:
Yes!